keyboards/bt66tech/rules.mk

```
@@ -0,0 +1,33 @@
+# MCU name
+MCU = STM32F103
+
+# GENERIC STM32F103C8T6 board - stm32duino bootloader
+OPT_DEFS = -DCORTEX_VTOR_INIT=0x2000
+MCU_LDSCRIPT = STM32F103x8_stm32duino_bootloader
+BOARD = STM32_F103_STM32DUINO
+
+DFU_ARGS = -d 1eaf:0003 -a2 -R
+DFU_SUFFIX_ARGS = -v 1eaf -p 0003
+
+# project specific files
+VPATH += keyboards/bt66tech/bluepill
+SRC =	keyboard.c
+
+#BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = yes	# Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	      # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes
+RGBLIGHT_ENABLE = yes
+WS2812_DRIVER = spi
+
+# LAYOUTS = 60_ansi
```

Yes please. 
```suggestion
LAYOUTS = 60_ansi
```

Though, just a heads up, adding/enabling this may cause Travis CI to fail on some keymaps.  That is okay. As long as the default keymaps compile, and any keymaps you add with the keyboard, it is fine if it fails.